CONTRIBUTING.md

```diff
@@ -71,7 +69,7 @@ There are many ways to get in touch with _The Turing Way_ team!
 
 ### Receiving Updates
 
-- [@turingway](https://twitter.com/turingway) on Twitter
+- [@turingway](https://twitter.com/turingway) on X/Twitter
```

Comment:
There was some good discussion about this [here](https://github.com/the-turing-way/the-turing-way/pull/3396). Would be good to be consistent.